### ISSUE TITLE
2 PSU rig mining Issue!!!!

### ISSUE THREAD

**Issue by chianghiong** [nicehash/NiceHashMiner#1115]
hi guy, I am using nicehash legacy.

My rig always restart if I set my afterburner power limit to 90, now I am running 80. Got fews times after I install the Gigabypte PSU (power limit 90), I saw 3 of the GPU stop working after few seconds

Also I am using NH to mine, so if I turn on Lyra2 my rig also will hang and restart it self, now I only can only Equihash and Neoscrypt.

May i know what is the problem?

My rig spec:
-Asus EX-B250-V7 x 1
-Intel G4400, 3.3 GHZ , 3MB Cache, LGA1151 ( BX80662G4400)
-Corsair ForceGT 120GB SSD x 1
-Corsair HX1200 x 1
-Gigabyte Xtreme Gaming 1200w x 1
-MSI GeForce GTX 1080ti x 6
-Corsair DDR4 2400MHz 4GB x 1

I connect Corsair HX1200 to motherboard,SSD, CPU, 3 riser and 3 GPU, and Gigabyte Xtreme Gaming 1200w to 3 riser and 3 GPU.

Anyone have this problem????

**Comment by pagaiba** [comment]
Are you overclocking the GPU's? Also, use the Corsair software and check how much current is it outputting to the system. You might be overloading it.

**Comment by chianghiong** [comment]
Yes,I am overclocking the GPU, I use the power meter that plug into the wall socket to check the wattage, when I first got 3x1080ti, I set the power to 100 and it work fine and it take to the maximum power which is around 960Watt. Then now when i install the 2nd PSU (Gigabyte Xtreme Gaming 1200W PSU), the power limit that I can set is only up to 80 which it only use 600plus Watt the 3 x1080ti. If I set to 90, it will hang and restart.
